src/graphql/data/fetchGraphQL.ts

```diff
@@ -35,10 +33,10 @@ const fetchQuery = (params: RequestParameters, variables: Variables): Promise<Gr
     query: params.text, // GraphQL text from input
     variables,
   })
-  const url = isNFT ? NFT_URL : TOKEN_URL
+  // TODO: When gating is removed from gql endpoint, we can remove the isNFT check and just use base headers
```

Comment:
Awesome! I think we can also simplify our env management/guarding logic, unless that PR wasn't merged yet (tbh, I don't remember 😅)

Reply:
That PR hasn't merged yet. That'll be the last one before launch